packages/calypso-analytics/src/utils/current-user.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Internal dependencies
+ */
+import hashPii from './hash-pii';
+
+/**
+ * Module variables
+ */
+let _currentUser: CurrentUser;
+
+export interface CurrentUserHashedPii {
+	ID: string;
+	username: string;
+	email: string;
+}
+
+export interface CurrentUser {
+	ID: number;
+	username: string;
+	email: string;
+	hashedPii?: CurrentUserHashedPii;
```

Comment:
Are there any cases where `hashedPii` can be missing? If we make it non-optional it would catch any cases where we forget to set.